test/kubemark/resources/manifests/addons/podsecuritypolicies-rbac/kube-dns-autoscaler-binding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
should all be `rbac.authorization.k8s.io/v1beta1` (with associated changes to the subject stanzas)